### PR TITLE
Add python3 to required system_packages

### LINKED ISSUE #12: Failed installing package python3-dev
**Describe the bug**
Unsatisfiable constraints for python3-dev dependency.

**To Reproduce**
1. Setup the Prerequisites from the [README](https://github.com/ericmatte/ad-media-lights-sync#prerequisites)
1. Start AppDaemon

**Error stack trace**

```
ERROR: unsatisfiable constraints:
  python3-3.8.2-r0:
    breaks: python3-dev-3.8.2-r1[python3=3.8.2-r1]
    satisfies: world[python3=3.8.2-r0]
[18:16:58] FATAL: Failed installing package python3-dev
[cont-init.d] appdaemon.sh: exited 1.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] 99-message.sh: executing... 
-----------------------------------------------------------
                Oops! Something went wrong.

 We are so sorry, but something went terribly wrong when
 starting or running this add-on.
 
 Be sure to check the log above, line by line, for hints.
-----------------------------------------------------------
```

**Environment versions:**
 - ad-media-lights-sync: 0.3.2
 - Home Assistant: 0.112.1
 - AppDaemon: 0.2.6

### PR DESCRIPTION
**Pull request by thundergreen**
Closes #12

Due to new python3-dev version python3 has to be updated

### CODE REVIEW

**Comment by ericmatte** [comment]
Hi @thundergreen, thank you for pointing that out. Indeed, we were missing the `python` dependency.

I'm not really sure why it was working before and now not anymore, but adding `python` to the `system_packages` fixes the problem, as you pointed out.

If you have more info on that, I would like to know the reason for that.

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ system_packages:
   - musl
   - make
   - g++
+  - python3=3.8.2-r1
```
Comment (ericmatte) [inline README.md]:
```suggestion
  - python3
```

I prefer not fixing the dependency versions.

I've tested without it, and it also fixes the problem.

**Review by ericmatte — APPROVED** [review]
Thank you!

**Comment by thundergreen** [comment]
it worked without python as python is instaled by default.. by adding pythn it'll update the python package i guess. Nice to help out :)

**Pull request merged**